server/server.js

```diff
@@ -1,13 +1,21 @@
-const express = require("express");
-const app = express();
-const cors = require("cors");
-require("dotenv").config({ path: "./config.env" });
+require("dotenv").config({ path: "./config.env" })
+
+const express = require("express")
+const cors = require("cors")
+const record = require("./record")
+const users = require("./users")
+const connect = require("./connect")
+
 const port = process.env.PORT || 5000;
+
+const app = express();
+
 app.use(cors());
 app.use(express.json());
-app.use(require("./record"));
+app.use(record);
```

Comment:
Is there a particular reason for calling the require and use functions separately other than for organization purpose?

Reply:
No real reason, just for consistency and organization.